mailgun.js

```diff
@@ -0,0 +1,173 @@
+const express = require('express');
+const nodemailer = require('nodemailer');
+const mg = require('nodemailer-mailgun-transport');
+ 
+const auth = {
+  auth: {
+    api_key: process.env.MAILGUN_API_KEY,
+    domain: process.env.MAILGUN_DOMAIN
+  },
+}
+ 
+const nodemailerMailgun = nodemailer.createTransport(mg(auth));
+
+async function start(toEmail) {
+  
+nodemailerMailgun.sendMail({
+  from: toEmail.creator_email,
+  to: toEmail.recipient_email, 
+  subject: 'Turntabl Poll',
+  'h:Reply-To': 'isaac.agyen@turntabl.io',
```

Comment:
investigate 'h:' - why isn't this just reply-to

Reply:
With the 'h:Reply-To each user gets a unique hash in the reply-to field so I know who is replying. Basically the reply-to looks like this:

"h:Reply-To" : "inboundaddress+hash@mydomain.com"

Based on official Mailgun API docs here [sending](https://documentation.mailgun.com/api-sending.html#) you can specify the Reply-to option, but it must be defined as h:X-My-Header: